docs/source/jit_language_reference.rst

```diff
@@ -0,0 +1,438 @@
+     - Supported
+     -
+   * - `6.10.1. Value comparisons <https://docs.python.org/3/reference/expressions.html#value-comparisons>`_
+     - Partially Supported
+     - Maybe None == None should throw an error since the ref says not to do it, list/dict equality
+   * - `6.10.2. Membership test operations <https://docs.python.org/3/reference/expressions.html#membership-test-operations>`_
+     - Partially Supported
+     - **iter** lookup not supported
+   * - `6.10.3. Identity comparisons <https://docs.python.org/3/reference/expressions.html#is-not>`_
+     - Supported
+     -
+   * - `6.11. Boolean operations <https://docs.python.org/3/reference/expressions.html#boolean-operations>`_
+     - Supported
+     -
+   * - `6.12. Conditional expressions <https://docs.python.org/3/reference/expressions.html#conditional-expressions>`_
+     - Partially Supported
+     - if + lambda expr not supported
+   * - `6.13. Lambdas <https://docs.python.org/3/reference/expressions.html#lambda>`_
+     - Not Supported
+     -
+   * - `6.14. Expression lists <https://docs.python.org/3/reference/expressions.html#expression-lists>`_
+     - Partially Supported
+     - iterable unpacking not supported
+   * - `6.15. Evaluation order <https://docs.python.org/3/reference/expressions.html#evaluation-order>`_
+     - Supported
+     -
+   * - `6.16. Operator precedence <https://docs.python.org/3/reference/expressions.html#operator-precedence>`_
+     - Supported
+     -
+   * - `7. Simple statements <https://docs.python.org/3/reference/simple_stmts.html#>`_
+     - Irrelevant
+     -
+   * - `7.1. Expression statements <https://docs.python.org/3/reference/simple_stmts.html#expression-statements>`_
+     - Supported
+     -
+   * - `7.2. Assignment statements <https://docs.python.org/3/reference/simple_stmts.html#assignment-statements>`_
+     - Partially Supported
+     -
+   * - `7.2.1. Augmented assignment statements <https://docs.python.org/3/reference/simple_stmts.html#augmented-assignment-statements>`_
+     - Partially Supported
+     - A bunch of these don't work, see "Delimiters"
+   * - `7.2.2. Annotated assignment statements <https://docs.python.org/3/reference/simple_stmts.html#annotated-assignment-statements>`_
+     - Supported
+     -
+   * - `7.3. The assert statement <https://docs.python.org/3/reference/simple_stmts.html#the-assert-statement>`_
+     - Partially Supported
+     - Message is not customized (it gets desugared into a ``RaiseException``\ )
+   * - `7.4. The pass statement <https://docs.python.org/3/reference/simple_stmts.html#the-pass-statement>`_
+     - Partially Supported
+     - String parser error with 1 line functions
+   * - `7.5. The del statement <https://docs.python.org/3/reference/simple_stmts.html#the-del-statement>`_
+     - Not Supported
+     -
+   * - `7.6. The return statement <https://docs.python.org/3/reference/simple_stmts.html#the-return-statement>`_
+     - Supported
```

Comment:
I think we can just say supported. i dont know how try ... finally is relevant here